apps/condo/domains/acquiring/schema/AcquiringIntegrationContext.js

```diff
@@ -124,19 +204,45 @@ const AcquiringIntegrationContext = new GQLListSchema('AcquiringIntegrationConte
     },
     hooks: {
         validateInput: async ({ resolvedData, context, addValidationError, existingItem }) => {
-            if ((resolvedData['status'] === CONTEXT_FINISHED_STATUS || resolvedData['status'] === CONTEXT_VERIFICATION_STATUS) &&
-                !resolvedData['deletedAt']) {
+            if (
+                (
+                    [CONTEXT_FINISHED_STATUS, CONTEXT_VERIFICATION_STATUS].includes(resolvedData['status'])
+                    || [CONTEXT_FINISHED_STATUS, CONTEXT_VERIFICATION_STATUS].includes(resolvedData['invoiceStatus'])
+                )
+                && !resolvedData['deletedAt']
+            ) {
                 const newItem = { ...existingItem, ...resolvedData }
                 const activeContexts = await ContextServerSchema.getAll(context, {
                     organization: { id: newItem['organization'] },
-                    status_in: [CONTEXT_FINISHED_STATUS, CONTEXT_VERIFICATION_STATUS],
+                    OR: [
+                        { status_in: [CONTEXT_FINISHED_STATUS, CONTEXT_VERIFICATION_STATUS] },
+                        { invoiceStatus_in: [CONTEXT_FINISHED_STATUS, CONTEXT_VERIFICATION_STATUS] },
+                    ],
                     deletedAt: null,
                     id_not: newItem['id'],
                 })
                 if (activeContexts.length) {
                     addValidationError(CONTEXT_ALREADY_HAVE_ACTIVE_CONTEXT)
                 }
             }
+
+            /*
+             vatPercent constraints:
+             __________________________________
+                       |  vatPercent values
+             taxRegime | 20% | 10% |  0% | null
+             ---------------------------------
+             general   |  +  |  +  |  +  |  +
+             simple    |  +  |  +  |  -  |  +
+             ----------------------------------
+             */
+            const nextData = { ...existingItem, ...resolvedData }
```

Comment:
It's usually named `newItem`. Can we rename it to stay consistent (non-critical btw 😄 )

Reply:
IMHO 'new' means new model, 'next' means new or updated.